cmd/resim/cli.go

```diff
@@ -1,23 +1,13 @@
 package main
 
 import (
-	"log"
-	"strings"
+	"os"
 
-	"github.com/resim-ai/api-client/cmd/resim/commands"
-	"github.com/spf13/viper"
+	cli "github.com/resim-ai/api-client/cmd/resim/wrapper"
 )
 
-const EnvPrefix = "RESIM"
-
+// This is the entry point for the CLI.
 func main() {
-	viper.SetEnvPrefix(EnvPrefix)
-	viper.AutomaticEnv()
-	// This confusingly-named function defines the mapping from CLI parameter key to environment variable.
-	// CLI parameters use kebab-case, and env vars use CAPITAL_SNAKE_CASE.
-	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
-	err := commands.Execute()
-	if err != nil && err.Error() != "" {
-		log.Fatal(err)
-	}
+	status := cli.MainWithExitCode()
```

Comment:
i couldn't figure out a better way to do this since testscript expects an exit code

Reply:
Could we just do the previous code (without the wrapper) but do `os.Exit(0)` if we succeed?

Reply:
I think it needs a function with int return type for the map (https://github.com/resim-ai/api-client/pull/41/files#diff-27efad63577dad7354dd1184010b168396a01c50546a6cebbc944ca9ef06fcb2R45)

 and cobra needs main()

but maybe I misunderstood the docs?